### PR TITLE
Bart configuration update

### PR DESCRIPTION
**Pull request by imarekkus**
Update to bart config.
image

Tests passed
image

### CODE REVIEW

**Comment by imarekkus** [comment]
Hey @ydshieh! It turned out that copy-pasting the "(with random weights)" text was causing the check_code_quality test failure. But unfortunately I don't know what is causing this ConnectionResetError: [Errno 104] while building PR Documentation

**Comment by ydshieh** [comment]
> Hey @ydshieh! It turned out that copy-pasting the "(with random weights)" text was causing the check_code_quality test failure.
Glad it works now 💯 Thank you!


> But unfortunately I don't know what is causing this ConnectionResetError: [Errno 104] while building PR Documentation
You can ignore this :-)

File: utils/documentation_tests.txt [hunk 1]
```diff
@@ -101,3 +101,4 @@ src/transformers/models/wavlm/modeling_wavlm.py
 src/transformers/models/whisper/modeling_whisper.py
 src/transformers/models/whisper/modeling_tf_whisper.py
 src/transformers/models/yolos/modeling_yolos.py
+src/transformers/models/bart/configuration_bart.py
```
Comment (ydshieh) [inline utils/documentation_tests.txt:104]: Could you put this line in a sorted order within this file 🙏

**Review by ydshieh — CHANGES_REQUESTED** [review]

File: utils/documentation_tests.txt (hunk 1 above)
Comment (imarekkus) [reply utils/documentation_tests.txt]: Sure!

**Review by ydshieh — APPROVED** [review]
Very cool, and thank you again @imarekkus 🔥 !

**Pull request merged**